src/higgsptpartonic.cpp

```diff
@@ -738,13 +739,22 @@ double HiggsDpTpartonic::deltapartonic(double pt, double nn, double zz) {
       // part of the xsec that is also calculated in this function.
       case (0):  // gg-channel
       {
+        // dominant contribution in small-x are the terms multiplied by uu. 
+        // specifically the logs (not Li2) dominate uu 
         double uu =
             0.5 * std::pow(log(uh / th), 2) + std::pow(M_PI, 2) / 3. -
             log(sh / MH2) * log(-th / MH2) - log(sh / MH2) * log(-uh / MH2) -
             log(-uh / MH2) * log(-th / MH2) + std::pow(log(MH2 / sh), 2) +
             std::pow(log(MH2 / (MH2 - th)), 2) +
             std::pow(log(MH2 / (MH2 - uh)), 2) + 2. * Li2(1. - MH2 / sh) +
             2. * Li2(MH2 / (MH2 - th)) + 2. * Li2(MH2 / (MH2 - uh));
+        uu =
+            0.5 * std::pow(log(uh / th), 2) + std::pow(M_PI, 2) / 3. -
+            log(sh / MH2) * log(-th / MH2) - log(sh / MH2) * log(-uh / MH2) -
+            log(-uh / MH2) * log(-th / MH2) + std::pow(log(MH2 / sh), 2) +
+            std::pow(log(MH2 / (MH2 - th)), 2) +
+            std::pow(log(MH2 / (MH2 - uh)), 2) + 2. * Li2(1. - MH2 / sh) +
+            2. * Li2(MH2 / (MH2 - th)) + 2. * Li2(MH2 / (MH2 - uh));
         double de = 1.5 * beta0 * (log(-MUR2 / th) + log(-MUR2 / uh)) +
```

Comment:
This can be removed. I just added it to trace the origin of the dominant terms, but I guess I never removed it afterwards.